plugin-gradle/src/main/java/com/diffplug/gradle/spotless/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.gradle.spotless;
+
+final class Constants {
```

Comment:
Which constants should go here?  Every constant?  Maybe it should be named SpotlessTaskStrings, and it can include MoreStrings?

Reply:
That all sounds reasonable to me, given that `Constants` and `MoreStrings.capitalize` together are only being used in `SpotlessTask*`, `SpotlessPlugin` and `PaddedCellTaskTest`. :)